Dockerfile

```diff
@@ -1,10 +1,10 @@
 # ========================
 # Build Stage
 # ========================
-FROM rust:1.87.0-alpine3.22 AS builder
+FROM rust:1.90.0-alpine3.22 AS builder
 
 # Build platform argument (x86_64 or aarch64) (default: x86_64)
-ARG TARGETARCH=x86_64
+ARG TARGETARCH
```

Comment:
can remove all `TARGETARCH`

Reply:
want to keep it now for debugging, will delete when we have a tested pipeline for building/pushing images to both registries.